README.md

```diff
@@ -3,7 +3,7 @@
 ## Table of Contents
   - [Check-ins](#check-ins)
   - [Organization](#plan-for-project-organization)
-  - [Approach To Code Design](#approach-to-code-design)
+  - [Approaches To Project Organization & Code Design](#approach-to-code-design)
```

Comment:
Can we group up the two Organization sections into the same link (as it was before) but just change the second organization section headline to a subheading?

Code design should still be its own link

Reply:
Sorry we weren't clearer in the slack!
We were talking about `Plan for Project Organization` and `Approaches to Project Organization` both being included under the one "Organization" link.